src/features/greeting/greeting.js

```diff
@@ -0,0 +1,46 @@
+const FETCH_MESSAGE_REQUEST = 'FETCH_MESSAGE_REQUEST';
+const FETCH_MESSAGE_SUCCESS = 'FETCH_MESSAGE_SUCCESS';
+const FETCH_MESSAGE_FAILURE = 'FETCH_MESSAGE_FAILURE';
+
+const apiURL = 'http://localhost:3000';
```

Comment:
Please make sure the Rails Server is running on this same URL and port or else you have to update this URL / Port